service/service.go

```diff
@@ -94,13 +95,27 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var operatorFramework *framework.Framework
+	{
+		frameworkConfig := framework.DefaultConfig()
+
+		frameworkConfig.Logger = config.Logger
+
+		operatorFramework, err = framework.New(frameworkConfig)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
 	var operatorService *operator.Service
 	{
 		operatorConfig := operator.DefaultConfig()
 		operatorConfig.AzureConfig = azureConfig
 		operatorConfig.Flag = config.Flag
 		operatorConfig.K8sClient = k8sClient
 		operatorConfig.Logger = config.Logger
+		operatorConfig.OperatorFramework = operatorFramework
+		operatorConfig.Resources = []framework.Resource{}
```

Comment:
Currently, the Resources collection is empty. I'll do a later PR for the Resource Group resource.